config.yml

```diff
@@ -0,0 +1,112 @@
+defaults:
+  index:
+    # Route prefix used to form custom routes on the form
+    #  'route.<routePrefix>.<namespace>'
+    routePrefix:                  'index'
+
+    # Name of task-graph table in azure table storage
+    indexedTaskTableName:         'IndexedTasks'
+    namespaceTableName:           'Namespaces'
+
+    # Azure account
+    azureAccount:                 !env AZURE_ACCOUNT
+
+    # Publish references and schemas
+    publishMetaData:              'false'
```

Comment:
Let's use `!env:bool PUBLISH_METADATA`
Then we don't have to do `cfg.index.publishMetaData === 'true'`

See: https://github.com/taskcluster/taskcluster-queue/blob/2ab0317576d9b7c6ca9b011a76ff36aaf7eaff3e/config.yml#L3